src/components/ContributionPlanBundleContributionPlans.js

```diff
@@ -11,6 +11,9 @@ import DeleteIcon from '@material-ui/icons/Delete';
 import { DEFAULT_PAGE_SIZE, ROWS_PER_PAGE_OPTIONS } from "../constants";
 import CreateContributionPlanBundleDetailsDialog from "../dialogs/CreateContributionPlanBundleDetailsDialog";
 import UpdateContributionPlanBundleDetailsDialog from "../dialogs/UpdateContributionPlanBundleDetailsDialog";
+import {
+    decodeId,
+} from "@openimis/fe-core";
```

Comment:
There is already an existing import from `@openimis/fe-core`. Move `decodeId` there to avoid duplication.

Reply:
Done